lib/matplotlib/tests/test_text.py

```diff
@@ -365,6 +365,10 @@ def test_annotation_negative_fig_coords():
 
 def test_text_stale():
     fig, (ax1, ax2) = plt.subplots(1, 2)
+    print(dict.__getitem__(plt.rcParams, "backend"))
+    print(plt.rcParams["backend"])
+    print(plt._backend_mod)
+    print(fig.canvas)
```

Comment:
We don't usually include print calls in the tests, do we?  Are these left over from debugging, or are they intentional?  If the latter, given that they are unusual, do they merit a comment?

Reply:
that was for debugging the failure on travis, removing it.